### ISSUE TITLE
[DataCap Application] <Public Dataset> - <OFS>

### ISSUE THREAD

**Issue by Wukk-Amy** [1475Notary/1475-Allocator#12]
### Data Owner Name

NOAA

### Data Owner Country/Region

United States

### Data Owner Industry

Environment

### Website

https://www.noaa.gov/

### Social Media Handle

amy

### Social Media Type

Slack

### What is your role related to the dataset

Data Preparer

### Total amount of DataCap being requested

5120TiB

### Expected size of single dataset (one copy)

416TiB

### Number of replicas to store

10

### Weekly allocation of DataCap requested

512TiB

### On-chain address for first allocation

f15nszarmic4zxwxqcvzxmrtsmrjfon6smffhba3q

### Data Type of Application

Public, Open Dataset (Research/Non-Profit)

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
For decades, mariners in the United States have depended on NOAA's Tide Tables for the best estimate of expected water levels. These tables provide accurate predictions of the astronomical tide (i.e., the change in water level due to the gravitational effects of the moon and sun and the rotation of the Earth); however, they cannot predict water-level changes due to wind, atmospheric pressure, and river flow, which are often significant.

The National Ocean Service (NOS) has the mission and mandate to provide guidance and information to support navigation and coastal needs. To support this mission, NOS has been developing and implementing hydrodynamic model-based Operational Forecast Systems.

This forecast guidance provides oceanographic information that helps mariners safely navigate their local waters. This national network of hydrodynamic models provides users with operational nowcast and forecast guidance (out to 48 – 120 hours) on parameters such as water levels, water temperature, salinity, and currents. These forecast systems are implemented in critical ports, harbors, estuaries, Great Lakes and coastal waters of the United States, and form a national backbone of real-time data, tidal predictions, data management and operational modeling.
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
NOMADS Production OFS Data (30 day rolling retention)
CO-OPS Operational OFS Data (Historical Retention)
New data notifications for NOS OFS Historical Retention
New data notifications for OFS, only Lambda and SQS protocols allowed
```

### Where was the data currently stored in this dataset sourced from

AWS Cloud

### If you are a data preparer. What is your location (Country/Region)

Singapore

### If you are a data preparer, how will the data be prepared? Please include tooling used and technical details?

```text
We have technology about lotus and venus. We can preparing data into car files and transfer to sp we have communicated. Then they finish the storage.
```

### Please share a sample of the data

```text
aws s3 ls --no-sign-request s3://noaa-ofs-pds/
aws s3 ls --no-sign-request s3://noaa-nos-ofs-pds/
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Yearly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China, North America, Europe

### How will you be distributing your data to storage providers

HTTP or FTP server, Shipping hard drives, Lotus built-in data transfer, Venus built-in data transfer

### How did you find your storage providers

Slack

### Please list the provider IDs and location of the storage providers you will be working with.

```text
f01844999-US
f02128256-CA
f01113378-CN
f01106276-CN
f02114994-CN
```

### How do you plan to make deals to your storage providers

Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by Wukk-Amy** [comment]
@1475Notary hello notary, please look at my application.

**Comment by 1475Notary** [comment]
@Wukk-Amy 
Hello Amy,

I want to ask you some questions.

Is this your first time to apply? Have you ever applied for datacap from another allocator?
Do SPs you work with support retrieval?
Do you have enough token for your plan?
For bookkeeping, I'll keep a record of our communications. Do you accept that?

**Comment by Wukk-Amy** [comment]
@1475Notary Yes sir, I am just joining in fil+. After read the rules of fil+, I knew that sp should support retrieval and I have talk with them about the rule. They have the technology. Sp have enough token to take the deals.
Yes, I accept it.

**Comment by 1475Notary** [comment]
@Wukk-Amy Please send your email to my slack (1475Notary) to confirm your identity.

**Comment by Wukk-Amy** [comment]
@1475Notary 
![image](https://github.com/user-attachments/assets/2c1b86c4-3e85-4390-b495-05db520ea2d4)
Please confirm it. Thank you.

**Comment by 1475Notary** [comment]
@Wukk-Amy ok. I saw it.

**Comment by Wukk-Amy** [comment]
@1475Notary Dear allocator, we've added two sps to take our deals. f03159626-US / f03157879-US

**Comment by 1475Notary** [comment]
@Wukk-Amy ok, keep your diligence.

**Comment by galen-mcandrew** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
Comparing the previous two reports, we can see that the retrieval success rate is improving, and we hope that you will continue your efforts!

**Comment by 1475Notary** [comment]
checker:manualTrigger

**Comment by 1475Notary** [comment]
Looks like this report doesn't turn out as well as the last round, please contact the SPs to talk about these results. The quota may need to be reduced this trache, so hopefully we'll see an improvement soon.

**Comment by Wukk-Amy** [comment]
@1475Notary As the report showed, we stop the cooperation with that's sps who do not give high retrieval success rate. When we send a little data to these sps, we'll check the retrieval results. After that, we changed the sp list in time. We will continue to improve the process.

Full sp list. 
f02046115-HK  (Have stopped sending data)
f0114153-HN  (Have stopped sending data)
f03214920-HK
f03159626-US  (Plan to stop sending data)
f03100008-GD  (Have stopped sending data)
f03161261-NY
f03157879-US
f03145504-HK  (Have stopped sending data)
f03136895-US  (Have stopped sending data)
f03189917-US

**Comment by 1475Notary** [comment]
@Wukk-Amy ok. I'll keep an eye on the next reports to see if you'll continue to be allocated datacap.

**Comment by filecoin-watchdog** [comment]
checker:manualTrigger
